gradle/dependencies.gradle

```diff
@@ -106,7 +106,7 @@ def test = [
     rxjava2                 : "io.reactivex.rxjava2:rxjava:2.1.2",
     commonsLang3            : "org.apache.commons:commons-lang3:3.8.1",
     commonsLang             : "commons-lang:commons-lang:2.6",
-    lombok                  : "org.projectlombok:lombok:1.18.22",
+    lombok                  : "org.projectlombok:lombok:1.18.24",
```

Comment:
Was gonna suggest we keep this pinned to whatever version of Lombok we use internally, but that's already not the case regarding the "patch" number. We might need to be a bit more cautions if we ever see Lombok 1.20.x 🙂

Reply:
Right, fair enough.  Will be careful when upgrading in the future